Packs/SpamhausFeed/Integrations/SpamhausFeed/SpamhausFeed.yml

```diff
@@ -0,0 +1,139 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: SpamhausFeed
+  version: -1
+configuration:
+- display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- defaultvalue: '240'
+  display: Fetch Interval
+  name: feedFetchInterval
+  required: true
+  type: 19
+- additionalinfo: Type of the indicator in the feed.
+  display: Indicator Type
+  name: indicator_type
+  required: true
+  type: 0
+- defaultvalue: F - Reliability cannot be judged
+  display: Reliability
+  name: feedInstanceReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- name: expiration
+  required: false
+  type: 17
+- additionalinfo: When selected, the exclusion list will be ignored for this feed.
+  display: Bypass exclusion list
+  name: skipExclusionList
+  required: false
+  type: 8
+- defaultvalue: feedInstanceReputationNotSet
+  display: Mark as reputation
```

Comment:
```suggestion
  display: Mark with reputation
```

Reply:
nevermind this is something that server should update for BYOI integrations that fetch indicators.